internal/third_party/kubernetes/flexvolume/flexvolume.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Instead of creating a folder for `internal/third_party/kubernetes`, let's just include the sources in a rook package where they're needed. For example, we have copyrights with "The Kubernetes Authors" in a bunch of generated code.

Reply:
Done.